scripts/generate-test-services.ts

```diff
@@ -33,7 +33,7 @@ const generatorConfigOData = {
   generateTypedocJson: false,
   generatePackageJson: false,
   generateCSN: false,
-  generateSdkMetadata: true,
+  generateSdkMetadata: false,
```

Comment:
This might be a quick fix.

Currently, the `test service version check` (the checks job instead of tests job) failed due to changed value of `version` and `generatedAt`, which should be dynamic.
There are some  other related issues. With option `true`, it will generate some api hub metadata and the files are not committed. However, the `.gitignore` does not contain the generated metadata files, which is confusing.
If the generated metadata has to be there for some reasons, we need to introduce additional logic to support static value for `version` and `generatedAt`.

Reply:
yes, it was a mistake from my side to set this to true.